tests/unit/worker/test_submission_worker.py

```diff
@@ -30,6 +39,32 @@ def setUp(self):
             aws_secret_access_key=os.environ.get("AWS_SECRET_ACCESS_KEY"),
             aws_access_key_id=os.environ.get("AWS_ACCESS_KEY_ID"),
         )
+        self.user = User(
+            username="someuser",
+            email="user@test.com",
+            password="secret_password",
+        )
+        self.challenge_host_team = ChallengeHostTeam(
+            team_name="Test Challenge Host Team", created_by=self.user
+        )
+        self.challenge = Challenge(
+            title="Test Challenge",
+            description="Description for test challenge",
+            terms_and_conditions="Terms and conditions for test challenge",
+            submission_guidelines="Submission guidelines for test challenge",
+            creator=self.challenge_host_team,
+            start_date=timezone.now() - timedelta(days=2),
+            end_date=timezone.now() + timedelta(days=1),
+            published=False,
+            enable_forum=True,
+            anonymous_leaderboard=False,
+            max_concurrent_submission_evaluation=200000,
```

Comment:
Please reduce this number to something reasonable.

Reply:
The default is 100000, so I thought to keep it 200000. Will 100 do?

Reply:
I think 100 should be fine.

Reply:
Done. Please see changes.